src/test/java/api/items/ItemApiExamples.java

```diff
@@ -1226,6 +1230,55 @@ public void canSearchItemsByLocation() throws Exception {
       is(mainLibraryItem.getString("id")));
   }
 
+  @Test
+  public void itemHasLastCheckInPropertiesWhenTheyAreSet() throws Exception {
+    JsonObject readingRoomItem = new JsonObject()
+      .put("id", UUID.randomUUID().toString())
+      .put("holdingsRecordId", createInstanceAndHolding().toString())
+      .put("materialTypeId", getDvdMaterialType())
+      .put("permanentLoanTypeId", getCanCirculateLoanType())
+      .put("permanentLocationId", getMainLibraryLocation())
+      .put("temporaryLocationId", getReadingRoomLocation());
+
+    JsonObject lastCheckInObj = new JsonObject();
+    lastCheckInObj.put(USER_ID_FIELD, UUID.randomUUID().toString());
+    lastCheckInObj.put(SERVICE_POINT_FIELD, UUID.randomUUID().toString());
+    lastCheckInObj.put(DATETIME_FIELD, DateTime.now().toString());
+
+    readingRoomItem.put(LAST_CHECK_IN_FIELD, lastCheckInObj);
+
+    itemsStorageClient.create(readingRoomItem);
+
+    JsonObject readingRoomItems = findItems("effectiveLocationId=" + getReadingRoomLocation());
+    JsonObject actualItem = readingRoomItems.getJsonArray("items").getJsonObject(0);
+    JsonObject actualLastCheckIn = actualItem.getJsonObject(LAST_CHECK_IN_FIELD);
+
+    assertThat(actualLastCheckIn.getString(DATETIME_FIELD), is(notNullValue()));
+    assertThat(actualLastCheckIn.getString(SERVICE_POINT_FIELD), is(notNullValue()));
+    assertThat(actualLastCheckIn.getString(USER_ID_FIELD), is(notNullValue()));
+
+  }
+
+  @Test
+  public void itemHasNoLastCheckInPropertiesWhenNotSet() throws Exception {
+    JsonObject readingRoomItem = new JsonObject()
+        .put("id", UUID.randomUUID().toString())
+        .put("holdingsRecordId", createInstanceAndHolding().toString())
+        .put("materialTypeId", getDvdMaterialType())
+        .put("permanentLoanTypeId", getCanCirculateLoanType())
+        .put("permanentLocationId", getMainLibraryLocation())
+        .put("temporaryLocationId", getReadingRoomLocation());
+
+    itemsStorageClient.create(readingRoomItem);
+
+    JsonObject readingRoomItems = findItems("effectiveLocationId=" + getReadingRoomLocation());
+
+    JsonObject actualItem = readingRoomItems.getJsonArray("items").getJsonObject(0);
+    assertThat(actualItem, is(notNullValue()));
+    assertThat(actualItem.getJsonObject(LAST_CHECK_IN_FIELD), is(nullValue()));
+  }
+
+
```

Comment:
Could you please remove this extra line?

Reply:
Yep